config/env/development.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     }
   },
   app: {
-    title: defaultEnvConfig.app.title + ' - Development Environment'
+    title: defaultEnvConfig.app.title + ' - Development Environment',
+    domain: 'localhost:3000'
```

Comment:
Instead of hardcoding port 3000, it is possible have it default to the port specified in the config file?